### PR TITLE
Add possibility to disable collapsible attributions from Source

### LINKED ISSUE #8621: Contradiction between collapsible docs and OSM source
the OSM() source has a default behavior of hidding the attribution behind a button in the bottom right corner even on large screens (see it here https://openlayers.org/en/latest/examples/simple.html )

On https://openlayers.org/en/latest/apidoc/module-ol_control_Attribution-Attribution.html, it reads

> If you use an OSM source, [collapsible] should be set to false — see href="http://www.openstreetmap.org/copyright">www.openstreetmap.org/copyright OSM Copyright —

Shouldn't the OSM source create an Attribution object with collapsible set to false on its own, instead of providing only a copyright string ?

### PR DESCRIPTION
**Pull request by bobobobo**
This will fix #8621

### CODE REVIEW

File: src/ol/source/OSM.js [hunk 1]
```diff
@@ -76,7 +76,8 @@ class OSM extends XYZ {
       reprojectionErrorThreshold: options.reprojectionErrorThreshold,
       tileLoadFunction: options.tileLoadFunction,
       url: url,
-      wrapX: options.wrapX
+      wrapX: options.wrapX,
+      attributionsCollapsible: true
```
Comment (ahocevar) [inline src/ol/source/OSM.js]: Did you mean to initialize this with `false`?

File: src/ol/source/Source.js [hunk 2]
```diff
@@ -66,6 +67,12 @@ class Source extends BaseObject {
      */
     this.attributions_ = this.adaptAttributions_(options.attributions);
 
+    /**
+     * @private
+     * @type {boolean}
+     */
+    this.attributionsCollapsible_ = options.attributionsCollapsible;
```
Comment (ahocevar) [inline src/ol/source/Source.js]: You could default this to `true` here, then other subclasses do not need to be changed.

File: src/ol/source/Source.js [hunk 3]
```diff
@@ -120,6 +127,10 @@ class Source extends BaseObject {
     return this.attributions_;
   }
 
+  getAttributionsCollapsible() {
```
Comment (ahocevar) [inline src/ol/source/Source.js]: JSDoc comment with `@return` missing.

File: src/ol/source/OSM.js (hunk 1 above)
Comment (bobobobo) [reply src/ol/source/OSM.js]: Yes, this was a misstake, it is not initialized to false

File: src/ol/source/Source.js (hunk 3 above)
Comment (bobobobo) [reply src/ol/source/Source.js]: JSDoc comment added

**Comment by bobobobo** [comment]
I will try to add a unit test for the `Attribution` control

**Pull request merged**